tinynn/util/bn_rebuild.py

```diff
@@ -0,0 +1,76 @@
+import typing
+
+import torch
+import torch.nn as nn
+import copy
+
+
+class ConvBnForward(nn.Module):
```

Comment:
If this is limited to Conv2d, then it should be named as `ConvBn2dForward`. But I guess it is simpler to just implement the support for Conv1d.